ogarniacz_mp3.py

```diff
@@ -76,6 +76,20 @@ def remove_duplicates(processing_dir_path):
                                 continue # skoro tego pliku nie ma, to nie ma co tutaj robić
 
 
+def adjust_bitrates(processing_dir_path):
+    for dir_path, sub_dirs, files in os.walk(processing_dir_path):
+        for file_name in files:
+            file_path = os.path.join(dir_path, file_name)
+            if classes.MusicFile.is_music_file(file_path):
+                bitrate = classes.MusicFile.__getattr__("bit_rate")
```

Comment:
Przede wszystkim, potrzebujesz instancji `MusicFile`:
```
music_file = MusicFile(file_path)
```
po tym music file będzie miało atrybut `bit_rate` (jeśli pomyliłem nazwę, to sprawdź w słowniku), bo na klasie MusicFile jest funkcja `__getattr__`.